ic-os/components/setupos-scripts/devices.sh

```diff
@@ -3,9 +3,12 @@
 set -o nounset
 set -o pipefail
 
+source /opt/ic/bin/config.sh
+source /opt/ic/bin/functions.sh
+
 SHELL="/bin/bash"
 PATH="/sbin:/bin:/usr/sbin:/usr/bin"
-CONFIG_DIR="/var/ic/config"
+CONFIG_DIR="/config"
```

Comment:
This means we'll be passing ahead the un-sanitized config, right?

Reply:
Correct, but I just added better normalization within config_map_from_path, so now when config.ini is read by SetupOS or HostOS, it will apply the same normalization we were doing before. And I added more malformed data to the unit tests.